bin/elembio_find_id_run

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib ( -d "$Bin/../lib/perl5" ? "$Bin/../lib/perl5" : "$Bin/../lib" );
+use English qw(-no_match_vars);
+use Readonly;
+use Pod::Usage;
+use Getopt::Long;
+
+use npg_tracking::Schema;
+use Monitor::Elembio::RunFolder;
+
+our $VERSION = '0';
+
+sub get_log_config {
+  my $level = shift;
+  my @config_lines = (
+    "log4perl.logger = $level, A1",
+    "log4perl.logger.Monitor.Elembio.RunFolder = $level, A1",
+    'log4perl.appender.A1 = Log::Log4perl::Appender::Screen',
+    'log4perl.appender.A1.layout = Log::Log4perl::Layout::PatternLayout',
+    'log4perl.appender.A1.layout.ConversionPattern = %d %-5p %c - %m%n',
+    'log4perl.appender.A1.utf8 = 1',
+    '# Prevent duplicate messages with a non-Log4j-compliant Log4perl option',
+    'log4perl.oneMessagePerAppender = 1',
+  );
+  return join "\n", @config_lines;
+}
+
+main();
+
+sub main {
+  my $runfolder_path;
+  my $log4perl_config;
+  my $debug;
+  my $verbose;
+  GetOptions (
+    'help'                        => sub {
+      pod2usage(-verbose => 2, -exitval => 0);
+    },
+    '--runfolder_path|runfolder-path=s'   => \$runfolder_path,
+    'logconf=s'                           => \$log4perl_config,
+    'debug'                               => \$debug,
+    'verbose'                             => \$verbose,
+  );
+
+  if ($log4perl_config) {
+    Log::Log4perl::init($log4perl_config);
+  } else {
+    my $log_config = get_log_config('ERROR');
+    if ($debug) {
+      $log_config = get_log_config('DEBUG');
+    } elsif ($verbose) {
+      $log_config = get_log_config('INFO');
+    }
+    Log::Log4perl::init(\$log_config);
+  }
+  my $log = Log::Log4perl->get_logger('main');
+
+  my $monitor = Monitor::Elembio::RunFolder->new( runfolder_path      => $runfolder_path,
+                                                  npg_tracking_schema => npg_tracking::Schema->connect());
+  $log->info('Run folder of interest: ' . $runfolder_path);
+  $log->debug('Current status of the run: ' . $monitor->tracking_run()->current_run_status_description);
```

Comment:
tracking_run is going to create a run if it does not exist, isn't it? Not what we want here.